suite-common/suite-config/src/settings.ts

```diff
@@ -1,5 +1,5 @@
 export const settingsCommonConfig = {
     MAX_ACCOUNTS: 10,
     FRESH_ADDRESS_LIMIT: 20,
-    TXS_PER_PAGE: 25,
+    TXS_PER_PAGE: 10,
```

Comment:
leave a PR comment explaining why we leave it at 10 temporarily and when we wil change it